etc/profile-m-z/syncthing.profile

```diff
@@ -0,0 +1,116 @@
+# Firejail profile for syncthing
+# Description: File synchronization using public networks
+# This file is overwritten after every install/update
+
+##quiet
+# Persistent local customizations
+include syncthing.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.local/state/syncthing
+# NOTE: this assumes a ~/Sync directory to be shared by default. Maybe leave a requirement to edit local file to set dirs?
+noblacklist ${HOME}/Sync
+
+# NOTE: will cause WARNING: Failed to lower process
+#       priority: set I/O priority: operation not permitted
+#       So, we try to preemptively set it here:
+nice 2
+
+# no allow-*.inc
+
+##blacklist PATH
+# Disable Wayland
+blacklist ${RUNUSER}/wayland-*
+# Disable RUNUSER (cli only; supersedes Disable Wayland)
+blacklist ${RUNUSER}
+# Remove the next blacklist if your system has no /usr/libexec dir,
+# otherwise try to add it.
+blacklist /usr/libexec
+
+# disable-*.inc includes
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+#include disable-write-mnt.inc # we set disable-mnt
+#include disable-x11.inc # this causes an error...
+include disable-xdg.inc
+
+
+mkdir ${HOME}/.local/state/syncthing
+whitelist ${HOME}/.local/state/syncthing
```

Comment:
Is this program intended to be executed at the user or system level?

Does the `syncthing` executable act as the client, server or both?

Reply:
Answer is both for both questions.

Ideally it is a user service. Most commonly it is system service. But upstream likes everyone to just run the program in a terminal and then minimize it.

Then it is a peer in a network which will connect to all sorts of things (announce servers, relay servers, other peers), while also exposing servers for LAN auto-discovery and to receive other peer's connections.